charts/consul/templates/connect-inject-deployment.yaml

```diff
@@ -236,24 +236,12 @@ spec:
                 {{- end }}
                 {{- end }}
 
-                {{- if .Values.global.consulSidecarContainer }}
```

Comment:
These flags are no longer used

Reply:
The removal of these are actually in the PR I am working on right now as I missed them last week.

Reply:
Hopefully the rebase of main onto your PR should automatically clear some of that from your commits.